docs/structure-of-a-contract.rst

```diff
@@ -123,6 +123,16 @@ Structs are custom defined types that can group several variables (see
             uint vote;
         }
     }
+    
+If you want to create a new struct variable inside a function, 
+and then add push it to an array, then you should create it in 
```

Comment:
I would not mention whether the struct variable is created inside a function, but rather only mention the intention to add a new element to a struct array.